packages/native/src/components/message/Notification/index.tsx

```diff
@@ -68,33 +77,29 @@ export default function Notification({
             variant={"body"}
             fontWeight={"medium"}
             color={
-              color ||
-              (variant === "primary"
+              variant === "primary"
                 ? colors.palette.neutral.c00
-                : colors.palette.neutral.c80)
+                : colors.palette.neutral.c80
             }
-            mt={"2px"}
-            mb={"2px"}
+            mt={2}
```

Comment:
When you input a number, it actually references the index on the "space" property of the theme, doesn't it?
So in this case you're actually setting margin top to 4px, is that intentional?

Reply:
Yes it's on purpose :ok_hand: